Update AUTHORS_RESUBMITTING_UNDER_LGPL_LICENSE.md

argilo opened the pull request:
I, Clayton Smith, hereby resubmit all my contributions to the VOLK project and repository under the terms of the LGPL-3.0-or-later. My GitHub handle is argilo. My email address used for contributions is: argilo@gmail.com.

I hereby agree that contributions made by me in the past, to previous versions of VOLK, may be re-used for inclusion in VOLK 3. I understand that VOLK 3 will be relicensed under LGPL-3.0-or-later.

michaelld reviewed (changes requested):
@argilo Thanks for your PR. Unfortunately, it is not sufficient to add your name to the relicense list. You also need to fill out the "[suggested text](https://github.com/gnuradio/volk/issues/492)" (bottom of the main description) to make it absolutely clear that you re-submit your contributions under LGPL. You can do this in the commit message / title or the PR message / title (though the message is preferred). Thanks!

argilo committed:
Update AUTHORS_RESUBMITTING_UNDER_LGPL_LICENSE.md

I, Clayton Smith, hereby resubmit all my contributions to the VOLK 
project and repository under the terms of the LGPL-3.0-or-later. My 
GitHub handle is argilo. My email address used for contributions is: 
argilo@gmail.com.

I hereby agree that contributions made by me in the past, to previous 
versions of VOLK, may be re-used for inclusion in VOLK 3. I understand 
that VOLK 3 will be relicensed under LGPL-3.0-or-later.

argilo commented:
Sorry about that. The commit message & PR description are now updated.
